webview/src/experiments/components/table/body/ExperimentStatusIndicator.tsx

```diff
@@ -21,10 +21,41 @@ type ExperimentStatusIndicatorProps = {
   id: string
 }
 
+const OnRemote: React.FC<{ id: string; isStudioConnected: boolean }> = ({
+  id,
+  isStudioConnected
+}) => {
+  if (isStudioConnected) {
+    return (
+      <CellHintTooltip
+        tooltipContent={'Experiment on remote\nClick to copy Studio link'}
+      >
+        <div
+          className={styles.upload}
+          {...clickAndEnterProps(() => copyStudioLink(id))}
+        >
+          <Icon
+            aria-label="copy-experiment-link"
```

Comment:
Technically, aria labels should be human-readable accessibility wise. For example, "Copy Experiment Link".